### ISSUE TITLE
time-grid imports day-grid but does not declare it as a dependency

### ISSUE THREAD

**Issue by i-am-gizm0** [vkurko/calendar#71]
I'm trying to add this calendar to a new SvelteKit project and have been running into a few issues (I'll open multiple because they're not all the same).

I'm trying to set up the most basic example but I'm running into problems:
```svelte
<script lang="ts">
    import Calendar from '@event-calendar/core';
    import TimeGrid from '@event-calendar/time-grid';

    let plugins = [TimeGrid];
    let options = {
        view: 'timeGridWeek',
        events: []
    };
</script>

<Calendar {plugins} {options} />
```
I installed `@event-calendar/core` and `@event-calendar/time-grid` (both v0.14.1)
When I load this page, the server throws an error:
> Cannot find module '@event-calendar/day-grid' imported from '(project directory)/node_modules/@event-calendar/time-grid/src/all-day/Week.svelte'

After installing `@event-calendar/day-grid` this error goes away but I wasn't expecting that to be a problem (glancing at the code, maybe `prepareEventChunks` should be moved from `day-grid` to `common`)

**Comment by vkurko** [comment]
Hi @i-am-gizm0 

Thank you for drawing my attention to this shortcoming. Of course, I will move `prepareEventChunks` to the `common` package.

**Comment by vkurko** [comment]
This should be fixed in v0.14.2. Please check.

**Comment by vkurko** [comment]
I hope I can close this issue.
